### ISSUE TITLE
Conscise list view in home feed

### ISSUE THREAD

**Issue by flexdinesh** [thepracticaldev/dev.to#1138]
Dev has grown tremendously and along with it, the number of blogs posts and topics a day as well. Back in the day, my feed usually used to have more posts that I'd prefer to open than today. This is mostly because we have so many types of posts these days (discussions, etc) and great many number of contributors. Back in the day, I will have scrolled though my entire feed and opened all the posts I wanted to read in a few minutes. But now, just going through all the posts is overwhelming as I have to scroll a lot and find significantly less number of posts I want to open.

I would like to have a list view option where the posts in the feed are listed in a concise view as in HN or reddit. This will give me the ability to skim through all the posts without having to scroll much.

**TLDR: _Concise feed view to fit in more posts in the window so the user doesn't have to scroll much._**

Note: This is more of a UX improvement than a feature.
